src/frontend/static/icons/Hipster_DownArrow.svg

```diff
@@ -1 +1,63 @@
-<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" data-name="Layer 1" viewBox="0 0 10 6"><defs><style>.cls-1{fill:#605f64}</style></defs><title>Hipster</title><polygon points="5 6 0 1 1 0 5 4 9 0 10 1 5 6" class="cls-1"/></svg>
```

Comment:
The `Hipster_DownArrow.svg` file is the image used as the down chevron/arrow in drop-downs (e.g., see top-right currency drop-down). It looks different now.